src/components/pages/blog-post/blog-post.tsx

```diff
@@ -28,6 +28,15 @@ const PanelContainer = styled.div`
   }
 `;
 
+const StyledArticle = styled.article`
+  display: grid;
```

Comment:
Ah, here you introduce the grid again. This shouldn't be necessary if you don't remove the grid from the parent.